events/events.go

```diff
@@ -81,22 +81,17 @@ func WatchEventStatus(ctx context.Context, client kubernetes.Interface, namespac
 	}
 	kch := w.ResultChan()
 	ch := make(chan *EventStatus, 2)
+	initialTimestamp := metav1.Now()
 
 	go func() {
 		defer close(ch)
-		// seen is used to drop duplicate updates
```

Comment:
do we not need de-deduping?

Reply:
We don't need deduping for events as there are no duplicates.